CHANGELOG.md

```diff
@@ -2,10 +2,20 @@
 
 ## v0.10.0
 
+### Technical improvements
+
 - Added new `recorder` module to build `Session Recorder` app
 - Added new `common` module to share code between `app` and `recorder` modules
 - Improved `VisionView` lifecycle management
 
+### User experience
+
+- MapboxVision v0.10.0
```

Comment:
Don't think it's really worth to mention.
Maybe we can just mention it in `technical` section

Reply:
Indeed will remove it